Chatbot.js

```diff
@@ -16,9 +16,22 @@ const portfolioInfo = {
         "Login Authentication",
         "Movie Recommend"

```

Comment:
Remove this line " movie recommend"

Reply:
I have removed the "movie recommend" already, Thank you .